src/Broadcast/InternalState.php

```diff
@@ -44,14 +45,18 @@ final class InternalState
     private StatusInternal $status = StatusInternal::IDLING_BEFORE_GATHERING_PEERS;
     private DeferredCancellation $cancellation;
     private bool $cancelled = false;
+    /** Number of seconds to wait between each peer. */
+    private int $delay = 0;
     public function __construct(
         private int $broadcastId,
         private MTProto $API,
         private Action $action,
         private Filter $filter,
+        float $delay = 0,
```

Comment:
private readonly float $delay = 0.0

Reply:
if someone input negative numbers, then it will flood log